routes/questions.js

```diff
@@ -81,17 +81,17 @@ router.post("/search", async (req, res) => {
 });
 
 
-router.get("/:id", async (req, res) => {
+/* router.get("/:id", async (req, res) => {
```

Comment:
There is an issue at this moment with the handlbar  of this route and server gets crash and This "/:id" and "/new".  this  routes are conflicting each other. so have to comment out  /:id route at this moment to test the other routes